### PR TITLE
refactor: Change format of input to DynamicTable

### LINKED ISSUE #69: Rewrite input to DynamicTable

### PR DESCRIPTION
**Pull request by ingeridhellen**
## Why is this pull request needed?

This pull request is needed to make code easier to read and reuse

## What does this pull request change?

Rewrites the input format for DynamicTable to better match a table structure.

## Issues related to this change:

Closes #69

### CODE REVIEW

**Review by mariuswinger — APPROVED** [review]
So much cleaner!

File: web/src/components/feature/PhaseTable.tsx [hunk 1]
```diff
@@ -1,36 +1,20 @@
...
-      42.29256 *
-      multiFlashResponse.phaseValues[phase]['percentage'] *
-      multiFlashResponse.componentFractions['5'][index] *
-      200.59
-    tableInput['Mercury Flow (g/d)'].push(hgPhaseFlow.toString())
-  })
-  return tableInput
+): string[][] {
+  const phPhaseFlowFactor = cubicFeedFlow * 42.29256 * 200.59
+  const mercury = multiFlashResponse.componentFractions['5']
```
Comment (soofstad) [inline web/src/components/feature/PhaseTable.tsx:9]: Mistet noen kommentarer her

**Review by soofstad — APPROVED** [review]
Bra! Mer leselig :)

File: web/src/components/feature/PhaseTable.tsx (hunk 1 above)
Comment (ingeridhellen) [reply web/src/components/feature/PhaseTable.tsx:9]: Good point, added issue on fixing the problem indicated in the second comment. First comment is not needed due to a more descriptive variable name, though.

**Pull request merged**